src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-box {
+  display: flex;
+  position: relative;
+  margin: 20px 0;
+}
+
+.icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.searchInput {
+  display: inline-block;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

Comment:
Move this style to `body` selector

Reply:
These are styles for input. If I move them my layout will break

Reply:
ok, thanks =)